package.json

```diff
@@ -4,14 +4,10 @@
   "description": "A React Native module for iOS that provides Spotlight search functionality.",
   "main": "index.js",
   "types": "index.d.ts",
-  "files": [
```

Comment:
I guess this was removed because something was missing from the package - would you be able to add the file/folders instead of removing the list, or was there another reason?

Reply:
As I was using this as a forked repo in one of the projects and `plugins` folder was missing from the package after doing `yarn install`. So, for the time I just removed files.

I'll try to make this change by this week. @jdmunro